ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -30,4 +30,8 @@ tests:
   commands: make functests-on-ci
   openshift_installer_src:
     cluster_profile: gcp
-
+- as: e2e-aws
```

Comment:
I'd give it a more descriptive name, like e2e-periodic or something. Also, since it's running on gcp I'd remove aws from the name.

Reply:
oh, copy paste failure there. thanks for pointing that out. It's not even running on aws, ha